usaspending_api/etl/tests/integration/test_load_transactions_in_delta.py

```diff
@@ -804,8 +803,436 @@ def test_inserts_and_deletes(
                 },
             ]
         )
+        assert equal_datasets(expected_award_id_lookup, delta_data, "")
 
-        # Verify transaction_id_lookup table
-        query = "SELECT * FROM int.award_id_lookup ORDER BY id, transaction_unique_id"
+
+class TestTransactionFabs:
+
+    expected_initial_transaction_fabs = [
+        {
+            "afa_generated_unique": "award_assist_0001_trans_0001",
+            "is_active": True,
+            "published_fabs_id": 1,
+            "transaction_id": 1,
+            "unique_award_key": "award_assist_0001",
+            "updated_at": datetime(year=2022, month=10, day=31),
+        },
+        {
+            "afa_generated_unique": "award_assist_0002_trans_0001",
+            "is_active": True,
+            "published_fabs_id": 2,
+            "transaction_id": 3,
+            "unique_award_key": "award_assist_0002",
+            "updated_at": datetime(year=2022, month=10, day=31),
+        },
+        {
+            "afa_generated_unique": "award_assist_0002_trans_0002",
+            "is_active": True,
+            "published_fabs_id": 3,
+            "transaction_id": 5,
+            "unique_award_key": "award_assist_0002",
+            "updated_at": datetime(year=2022, month=10, day=31),
+        },
+        {
+            "afa_generated_unique": "award_assist_0003_trans_0001",
+            "is_active": True,
+            "published_fabs_id": 4,
+            "transaction_id": 7,
+            "unique_award_key": "award_assist_0003",
+            "updated_at": datetime(year=2022, month=10, day=31),
+        },
+        {
+            "afa_generated_unique": "award_assist_0003_trans_0002",
+            "is_active": True,
+            "published_fabs_id": 5,
+            "transaction_id": 8,
+            "unique_award_key": "award_assist_0003",
+            "updated_at": datetime(year=2022, month=10, day=31),
+        },
+    ]
+
+    transaction_fabs_compare_fields = expected_initial_transaction_fabs[0].keys()
+
+    @mark.django_db(transaction=True)
+    def test_no_initial_run(
+        self, spark, s3_unittest_data_bucket, hive_unittest_metastore_db, populate_initial_postgres_data
+    ):
+        load_initial_delta_tables(spark, s3_unittest_data_bucket)
+
+        with raises(pyspark.sql.utils.AnalysisException, match="Table or view not found: int.transaction_fabs"):
+            call_command("load_transactions_in_delta", "--etl-level", "transaction_fabs")
+
+    @staticmethod
+    def initial_load_no_transaction_id_lookup(spark, s3_unittest_data_bucket):
+        TestInitialRun.initial_run(spark, s3_unittest_data_bucket)
+        call_command("load_transactions_in_delta", "--etl-level", "transaction_fabs")
+
+    @mark.django_db(transaction=True)
+    def test_initial_load_no_transaction_id_lookup(
```

Comment:
Can `test_initial_load_no_transaction_id_lookup` and `test_inserts_updates_deletes_no_transaction_id_lookup` be combined? You can still check the results of initial call match the expected output before making the inserts/updates/deletes. Same for the other three sets of `test_initial_x`/`test_inserts_updates_deletes_x`

Reply:
It would be cool if you could sneak in the scenario from `test_no_initial_run` into this test too, so we can cutback on the total number of Spark jobs

Reply:
ok